src/unitxt/operators.py

```diff
@@ -648,7 +653,8 @@ class ShuffleFieldValues(FieldOperator):
 
     def process_value(self, value: Any) -> Any:
         res = list(value)
-        get_random().shuffle(res)
+        random_generator = new_random_generator(sub_seed=str(hash(str(res))))
```

Comment:
Same comment here it will have exactly the same augmentation per given value.

Reply:
Same comment as above for me too.  Do we want an additional element in the list to change how all elements are shuffled?

Reply:
same decision, keep dependence on the value
@yoavkatz was your intention that even depending on the entire list is too much?